validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+| 9 | Attempt volume creation as read-only user | | Operation blocked with appropriate error message |
+| 10 | Verify RBAC consistency across both UIs | | Permissions enforced consistently |
+
+### Volume Creation Through Rancher Workloads
+
+Verify automatic volume provisioning and attachment when deploying workloads with Longhorn storage through Rancher.
+
+| Step Number | Action | Data | Expected Result |
+|------|--------|--------|-----------------|
+| 1 | Navigate to "Workloads" in Rancher cluster view | | Workloads section displays |
+| 2 | Click "Create" to deploy new workload | | Workload creation form opens |
+| 3 | Configure workload with name `test-longhorn-workload` and nginx image | | Basic workload configuration set |
+| 4 | Click "Add Volume" → "Add a new persistent volume (claim)" | | Volume configuration dialog opens |
+| 5 | Configure PVC: name=`test-pvc`, storageClass=`longhorn`, size=`1Gi`, accessMode=`ReadWriteOnce` | | PVC settings configured |
+| 6 | Set mount path to `/data` | | Mount path configured |
+| 7 | Deploy the workload | | Workload deployment begins |
+| 8 | Verify workload pod reaches Running state | | Pod shows "Running" status |
+| 9 | Check Longhorn UI for automatic volume creation | | Volume appears in Longhorn UI as "Attached" |
+| 10 | Exec into pod and write test data to `/data` | | Data write succeeds |
+| 11 | Delete and recreate pod, verify data persistence | | Data persists across pod restart |
+
+### StatefulSet Volume Management
+
+Test Longhorn integration with StatefulSet persistent volumes including scaling operations and pod rescheduling.
+
+| Step Number | Action | Data | Expected Result                                   |
+|------|--------|--------|----------------------------------------------------|
+| 1 | Create StatefulSet manifest with volumeClaimTemplates using Longhorn storage class | | Manifest prepared with Longhorn PVC template       |
+| 2 | Deploy StatefulSet with 3 replicas via Rancher or kubectl | | StatefulSet deployment begins                      |
+| 3 | Verify each pod gets dedicated persistent volume | | 3 volumes created, each attached to respective pod |
+| 4 | Write unique test data to each pod's volume | | Data successfully written to all volumes           |
+| 5 | Scale StatefulSet up to 5 replicas | | Scaling operation succeeds                         |
+| 6 | Verify new pods get new dedicated volumes | | 2 additional volumes created and attached          |
+| 7 | Scale StatefulSet down to 2 replicas | | Scaling down completes                             |
+| 8 | Verify volumes remain available for future scaling | | Volumes persist in "Detached" state                |
+| 9 | Delete one pod to trigger rescheduling | | Pod rescheduled to different node                  |
+| 10 | Verify volume reattaches to rescheduled pod with data intact | | Data consistency maintained                        |
+
+### Longhorn Metrics in Rancher Monitoring
```

Comment:
would reword to `Rancher Monitoring Integration` or `Rancher Monitoring Interoperability`

Reply:
Done